app/javascript/highchart_theme.js

```diff
@@ -11,7 +11,7 @@ Highcharts.theme = {
     "rgba(243, 123, 29, 1)"
   ],
   chart: {
-    backgroundColor: '#32383e',
+    backgroundColor: '#262830',
```

Comment:
The background for the graphs isn't the same as the card background anymore.  Not sure if this was intended or not.

![chrome_2018-06-08_19-53-58](https://user-images.githubusercontent.com/7030913/41186986-b1a4c16a-6b55-11e8-93c1-712d9b413cf1.png)

Reply:
I think I must have fixed this in a recent commit -- this is what I see:

<img width="1200" alt="screenshot 2018-06-09 16 29 01" src="https://user-images.githubusercontent.com/438911/41196949-481378f6-6c02-11e8-9765-3904822e17b7.png">

Reply:
Oh woops, it's not fixed on the stats page, only the individual segment history graphs. Will fix.

Reply:
Looks like the backgrounds are different colors, might be from the striped table?